htdocs/commande/list_det.php

```diff
@@ -407,7 +407,7 @@
 	$sql .= ' LEFT JOIN '.MAIN_DB_PREFIX.'commandedet as pd ON c.rowid=pd.fk_commande';
 }
 if ($search_product_category > 0) {
-	$sql .= ' LEFT JOIN '.MAIN_DB_PREFIX.'categorie_product as cp ON cp.fk_product=pd.fk_product';
+	$sql .= ' LEFT JOIN '.MAIN_DB_PREFIX.'categorie_product as cp ON cp.fk_product=cdet.fk_product';
```

Comment:
Filter on categories must no more use the LEFT JOIN solution. It must use a solution without adding tables in the SELECT nor into the FROM.
All conditions must be into the WHERE. See example into projet/list.php after comment
// Search for tag/category ($searchCategoryProjectList is an array of ID)
Can you investigate to make changes into this direction (this is a goal to modify everywhere in dolibarr this way).

Reply:
@eldy it's done